### DISCUSSION TITLE
PgEdge platform on Kubernetes setup instructions / guide please

### DISCUSSION THREAD

**Discussion by Rammurthy5** [pgEdge/cli#56] [Q&A]
Hi pgEdge team, thanks for swift response to all my previous queries in the past!
I am now trying to configure PgEdge in Kubernetes cluster. May i ask if there is a setup guide to follow like there is one [here](https://docs.pgedge.com/platform/installing_pgedge/getting_started) for EC2, please?

**Comment by luss** [comment]
Hi Ram, depending what flavor of k8's you are trying to use it can be a beast...  (as you know)   We have a docker file that makes a container for you.  What else do you really need.  Perhaps you'd like to write a blog post, which we can help you promote, with your Kubernetic Journey.  My personal recommendation for learning more about container netwkring stuff is to use & learn the basics of docker engine (formerly call Docker Compose) for settting up a simple cluster.    Also, we have a CLI command `./nc cluster local-create` thatwill create and fully configure an N-Node cluster for you on localhost without using any containers at all.   In my situations for running a serious database...  cotainers are not worth their overhead and/or security risks.  They are inherently more complicated when you are trying to run a DB because containers were (and still are) optimized for stateless applications.
